File: 001-100/083/B.cpp
#include <bits/stdc++.h>
using namespace std;

int digitSum(int N) {
    int res = 0;
    while (N >= 1) {
        res += (N % 10);
        N /= 10;
    }
    return res;
}
int main() {
    int N, A, B;
    cin >> N >> A >> B;
    int ans = 0;
    for (int i = 1; i <= N; i++) {
        int sum = digitSum(i);
        if ((A <= sum) && (sum <= B)) {
            ans += i;
        }
    }
    cout << ans << endl;
    return 0;
}

File: 001-100/083/B.py
n, a, b = map(int, input().split())


def digit_sum(num):
    s = list(str(num))
    sum = 0
    for c in s:
        sum += int(c)
    return sum


ans = 0
for i in range(1, n + 1):
    if a <= digit_sum(i) <= b:
        ans += i
print(ans)
